temporal-sdk/src/main/java/io/temporal/internal/sync/ActivityExecutionContextImpl.java

```diff
@@ -72,6 +75,9 @@
   private final Lock lock = new ReentrantLock();
   private ScheduledFuture future;
   private ActivityCompletionException lastException;
+  private final ManualActivityCompletionClientFactoryImpl manualCompletionClientFactory;
```

Comment:
Does it need to be the Impl or the ManualActivityCompletionClientFactory interface would be enough?

Reply:
Interface is enough.